modules/logging-loki-storage-aws.adoc

```diff
@@ -25,3 +25,17 @@ $ oc create secret generic logging-loki-aws \
   --from-literal=access_key_secret="<aws_access_key_secret>" \
   --from-literal=region="<aws_region_of_your_bucket>"
 ----
+
+== AWS storage for STS enabled clusters
+
+If your cluster has STS enabled, the Cloud Credential Operator (CCO) supports short-term authentication using AWS tokens.
+
+You can create the Loki object storage secret manually by running the following command:
+[source,terminal,subs="+quotes"]
+----
+oc -n openshift-logging create secret generic "logging-loki-aws"
+--from-literal=bucketnames="<s3_bucket_name>"
+--from-literal=region="<bucket_region>"
```

Comment:
These instructions apply on any AWS STS OpenShift clusters. However according to [LOG-5294](https://issues.redhat.com/browse/LOG-5294) we need to set the `audience` field on ROSA/HCP clusters. @kabirbhartiRH Can you provide a sample for this one?

Reply:
Ah yes. I din't see the error issue on ROSA cluster but on Hypershift hosted cluster, yes. I think we can add a note to include in case the user faces `err="WebIdentityErr: failed to retrieve credentials\ncaused by: InvalidIdentityToken: Incorrect token audience\n\tstatus code: 400 issue`, they can add audience as an attribute on the secret like 

```
oc -n openshift-logging create secret generic "logging-loki-s3" \
--from-literal=bucketnames="<s3_bucket_name>" \
--from-literal=region="<bucket_region>" \
--from-literal=audience="<oidc_audience>" 
```

Reply:
> Ah yes. I din't see the error issue on ROSA cluster but on Hypershift hosted cluster, yes. I think we can add a note to include in case the user faces `err="WebIdentityErr: failed to retrieve credentials\ncaused by: InvalidIdentityToken: Incorrect token audience\n\tstatus code: 400 issue`, they can add audience as an attribute on the secret like
> 
> ```
> oc -n openshift-logging create secret generic "logging-loki-s3" \
> --from-literal=bucketnames="<s3_bucket_name>" \
> --from-literal=region="<bucket_region>" \
> --from-literal=audience="<oidc_audience>" 
> ```
> 
> In our case, oidc_audience is 'openshift'

This seems like something that should have been included in release notes as a Known Issue.